docs/external-classes.json

```diff
@@ -1,98 +1,50 @@
 [{
   "name": "Google\\Auth\\",
-  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.php"
+  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.html"
 }, {
   "name": "Google\\GAX\\",
-  "uri": "https://github.com/googleapis/gax-php/tree/master/src/%s.php"
+  "uri": "https://googleapis.github.io/gax-php/master/Google/GAX/%s.html"
```

Comment:
Hmm good point. I have just pushed a change so that `#` is used to redirect to the latest docs, but maybe this isn't what we want.

Parsing the composer file could work, but pinning to a specific GAX version doesn't seem quite right either. If our dev dependency is `"google/gax": "^0.20.0"` but we release a `0.20.1` version, we would probably want to point to that instead.

Similarly with the auth library, we depend on `^1.0` so it seems to make sense to point to `1.1` if it is available, because that is what a new user will get if they run `composer install`